platform/platform-commands/src/main/java/org/codice/ddf/commands/platform/ConfigStatusCommand.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.commands.platform;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collection;
+
+import org.apache.felix.gogo.commands.Command;
+import org.codice.ddf.configuration.store.ConfigurationStatusService;
+import org.fusesource.jansi.Ansi;
+import org.fusesource.jansi.Ansi.Attribute;
+
+@Command(scope = PlatformCommands.NAMESPACE, name = "configstatus", description = "Lists import status of configuration files.")
+public class ConfigStatusCommand extends PlatformCommands {
+
+    private String configDirectory;
+
+    static final String SUCCESSFUL_IMPORT_MESSAGE = "All config files in [%s] imported successfully.";
+
+    static final String FAILED_IMPORT_MESSAGE = "Failed to import file [%s]. Moved to [%s].";
+
+    private ConfigurationStatusService configStatusService;
+
+    public ConfigStatusCommand(ConfigurationStatusService connfigStatusService,
+            String configDirectory) {
+        this.configStatusService = connfigStatusService;
```

Comment:
It shouldn't affect anything, but there's a weird double-n thing going on with the connfigStatusService parameter